### PR TITLE
Update zh-cn translate for index.md

### PR DESCRIPTION
**Pull request by undownding**

### CODE REVIEW

**Review by laosb — APPROVED** [review]
looks good to me

**Review by Lellansin — APPROVED** [review]
LGTM

File: locale/zh-cn/index.md [hunk 1]
```diff
@@ -15,7 +15,10 @@ labels:
...
   version-schedule-prompt: 可参考
   version-schedule-prompt-link-text: LTS 日程
+  newsletter: true
+  newsletter-prefix: 注册
+  newsletter-postfix: ", Node.js 官方的新闻周报。"
 ---
 
 Node.js®是一个基于[Chrome V8 引擎](https://developers.google.com/v8/)的 JavaScript 运行时。
-Node.js 使用高效、轻量级的事件驱动、非阻塞 I/O 模型。Node.js 之生态系统是目前最大的开源包管理系统。
+Node.js 使用高效、轻量级的事件驱动、非阻塞 I/O 模型。Node.js 的包管理器，[npm](https://www.npmjs.com/)，是目前世界上最大的开源项目管理包管理工具。
```
Comment (JacksonTian) [inline locale/zh-cn/index.md]: 开源项目管理包管理工具。

Comment (undownding) [reply locale/zh-cn/index.md]: 这就改

Comment (undownding) [reply locale/zh-cn/index.md]: 这句话斟酌了半天，该如何翻译比较合适？

package ecosystem 在这里结合上下文来看指的是 npm 本身还是整个以 npm 为首的生态环境？

Comment (laosb) [reply locale/zh-cn/index.md]: 我感觉可以直接翻译为「包生态系统」？

File: locale/zh-cn/index.md [hunk 2]
```diff
@@ -15,7 +15,10 @@ labels:
...
   version-schedule-prompt: 可参考
   version-schedule-prompt-link-text: LTS 日程
+  newsletter: true
+  newsletter-prefix: 注册
+  newsletter-postfix: ", Node.js 官方的新闻周报。"
 ---
 
 Node.js®是一个基于[Chrome V8 引擎](https://developers.google.com/v8/)的 JavaScript 运行时。
-Node.js 使用高效、轻量级的事件驱动、非阻塞 I/O 模型。Node.js 之生态系统是目前最大的开源包管理系统。
+Node.js 使用高效、轻量级的事件驱动、非阻塞 I/O 模型。Node.js 的包生态系统，[npm](https://www.npmjs.com/)，是目前世界上最大的开源库生态系统。
```
Comment (undownding) [inline locale/zh-cn/index.md]: 参考了原文发现本身就是上下文不连贯，还是以尊重原文为主。

@laosb

File: locale/zh-cn/index.md [hunk 3]
```diff
@@ -15,7 +15,10 @@ labels:
   api: API 文档
   version-schedule-prompt: 可参考
   version-schedule-prompt-link-text: LTS 日程
+  newsletter: true
+  newsletter-prefix: 注册
+  newsletter-postfix: ", Node.js 官方的新闻周报。"
```
Comment (XadillaX) [inline locale/zh-cn/index.md:20]: why not use '，' to instead of ','?

File: locale/zh-cn/index.md [hunk 4]
```diff
@@ -15,7 +15,10 @@ labels:
...
 ---
 
 Node.js®是一个基于[Chrome V8 引擎](https://developers.google.com/v8/)的 JavaScript 运行时。
```
Comment (XadillaX) [inline locale/zh-cn/index.md]: Maybe we should add a space after '&reg;'.

File: locale/zh-cn/index.md (hunk 2 above)
Comment (XadillaX) [inline locale/zh-cn/index.md]: Maybe we can use '它的包生态系统'?

Comment (XadillaX) [reply locale/zh-cn/index.md]:
```markdown
它的包生态系统 [npm](...) 是目前实际上最大的开源库生态系统。
```

**Review by XadillaX — COMMENTED** [review]
Some nits.

Refs: https://github.com/sparanoid/chinese-copywriting-guidelines

File: locale/zh-cn/index.md (hunk 4 above)
Comment (XadillaX) [reply locale/zh-cn/index.md]: Maybe we should add a space after '于' too.

File: locale/zh-cn/index.md (hunk 3 above)
Comment (undownding) [reply locale/zh-cn/index.md:20]: 注册 Node.js Everywhere,  Node.js 官方的新闻周报。

注册 Node.js Everywhere，Node.js 官方的新闻周报。

I think the frist one is better : )

**Comment by undownding** [comment]
@XadillaX All nits has fixed. Please review.

File: locale/zh-cn/index.md (hunk 2 above)
Comment (laosb) [reply locale/zh-cn/index.md]: 嗯有道理

File: locale/zh-cn/index.md (hunk 3 above)
Comment (laosb) [reply locale/zh-cn/index.md:20]: @undownding 我认为这属于句中的分隔符，逗号前后尽管都是英文，但是是两个专有名词，分属于两个子句，所以应当采用全角逗号。

Comment (XadillaX) [reply locale/zh-cn/index.md:20]: +1 with @laosb.

Refs: https://github.com/sparanoid/chinese-copywriting-guidelines#數字與單位之間需要增加空格

File: locale/zh-cn/index.md [hunk 5]
```diff
@@ -15,7 +15,10 @@ labels:
...
   version-schedule-prompt-link-text: LTS 日程
+  newsletter: true
+  newsletter-prefix: 注册
+  newsletter-postfix: ", Node.js 官方的新闻周报。"
 ---
 
-Node.js®是一个基于[Chrome V8 引擎](https://developers.google.com/v8/)的 JavaScript 运行时。
-Node.js 使用高效、轻量级的事件驱动、非阻塞 I/O 模型。Node.js 之生态系统是目前最大的开源包管理系统。
+Node.js® 是一个基于 [Chrome V8 引擎](https://developers.google.com/v8/)的 JavaScript 运行时。
+Node.js 使用高效、轻量级的事件驱动、非阻塞 I/O 模型。它的包生态系统，[npm](https://www.npmjs.com/)，是目前世界上最大的开源库生态系统。
```
Comment (yorkie) [inline locale/zh-cn/index.md:24]: 建议不要加最后最大，避免引起其他不必要的问题～

**Review by yorkie — CHANGES_REQUESTED** [review]

File: locale/zh-cn/index.md (hunk 5 above)
Comment (undownding) [reply locale/zh-cn/index.md:24]: Node.js' package ecosystem, npm, is the largest ecosystem of open source libraries in the world.

仔细斟酌的话原文是有地方是上下不连贯以及奇怪的，出于翻译的信达雅原则我认为还是应该以尊重原文为先。

原先文案有问题的话我们应该开 issue 提出，此 PR 应以“在保证尊重原意的基础上尽量表达得好”为审查标准。

File: locale/zh-cn/index.md (hunk 3 above)
Comment (undownding) [reply locale/zh-cn/index.md:20]: 我今天才看到，等下面得那个问题讨论得出结论后再一块修复。

**Comment by fhemberger** [comment]
@undownding Hi, what's the status of this PR? @yorkie Can you please set your review status as soon as everything is resolved?

**Comment by undownding** [comment]
@fhemberger need more discuss but no one join it

File: locale/zh-cn/index.md (hunk 5 above)
Comment (JacksonTian) [reply locale/zh-cn/index.md:24]: 尊重事实 > 尊重广告法

**Review by JacksonTian — APPROVED** [review]

**Comment by lpinca** [comment]
@yorkie can this land?

File: locale/zh-cn/index.md (hunk 3 above)
Comment (XadillaX) [reply locale/zh-cn/index.md:20]: the comma still need to be modified.

**Review by XadillaX — CHANGES_REQUESTED** [review]

**Comment by fhemberger** [comment]
Ok, as @yorkie doesn't reply and this has already been open for a month: I'd love to land this change. @undownding @XadillaX Could you please work out the last little nit so we can merge this?

If there are further changes/refinements required, please open a follow up PR.

**Review by yorkie — APPROVED** [review]

**Comment by fhemberger** [comment]
Ok, I'm merging this change. If there are still details to discuss, please open another PR.

Thanks for the update! :+1:

**Pull request merged**